hack/v1migration/phase1.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+
+set -e
+
+if [ $# != 1 ]; then
+	echo "USAGE: $0 [WORKDIR]"
+	exit 1
+fi
+
+WORKDIR=$1
+echo "Using workdir: $WORKDIR"
+
+oc project hive
+
+oc scale -n hive deployment.v1.apps/hive-operator --replicas=0
+oc scale -n hive deployment.v1.apps/hive-controllers --replicas=0
+oc scale -n hive deployment.v1.apps/hiveadmission --replicas=0
+
+# TODO: wait until no pods are running
+
+mkdir -p $WORKDIR/
+
+HIVE_TYPES=( checkpoints clusterdeployments clusterdeprovisionrequests clusterimagesets clusterprovisions clusterstates dnsendpoints dnszones hiveconfig selectorsyncidentityprovider selectorsyncset syncidentityprovider syncsetinstance syncset )
+for i in "${HIVE_TYPES[@]}"
+do
+	:
+	echo "Storing all ${i} in ${WORKDIR}/${i}.yaml"
+	oc get ${i}.hive.openshift.io -A -o json | jq '.items | .[] |
+		del(.status) |
```

Comment:
I don't think that we want to strip out the status. We will need it in some cases.

Reply:
I think I'm going to change it to backup everything but the last-applied annotation, and then strip out what we need with jq when submitting later.

Reply:
That would be my preference. I would like to preserve everything that we can.